application/frontend/config/main.php

```diff
@@ -2,64 +2,46 @@
 
 use Sil\PhpEnv\Env;
 
-/* Get frontend-specific config settings from ENV vars or set defaults. */
-$frontCookieSecure = Env::get('FRONT_COOKIE_SECURE', true);
-
-$sessionLifetime = 1800; // 30 minutes
-
-const UID_ROUTE_PATTERN = '<uid:([a-zA-Z0-9_\-]{32})>';
-
 return [
     'id' => 'app-frontend',
     'basePath' => dirname(__DIR__),
-    'bootstrap' => ['log', 'errorHandler'],
     'controllerNamespace' => 'frontend\controllers',
     'components' => [
+        'errorHandler' => [
+            'class' => 'frontend\web\ErrorHandler',
+        ],
         'user' => [
-            'identityClass' => 'common\models\User',
+            'identityClass' => 'common\models\ApiConsumer',
             'enableAutoLogin' => false,
             'enableSession' => false,
             'loginUrl' => null,
         ],
-        'session' => [
-            'cookieParams' => [// http://us2.php.net/manual/en/function.session-set-cookie-params.php
-                'lifetime' => $sessionLifetime,
-                'path' => '/',
-                'httponly' => true,
-                'secure' => $frontCookieSecure,
-            ],
-        ],
-        'log' => [
-
-        ],
-        'errorHandler' => [
-            'errorAction' => 'site/error',
-        ],
         'request' => [
-            'enableCsrfValidation' => false,
             'parsers' => [
-                'application/json' => 'yii\web\JsonParser',
+                'application/json' => 'yii\web\JsonParser', // required according to http://www.yiiframework.com/doc-2.0/guide-rest-quick-start.html#enabling-json-input
             ]
         ],
         'urlManager' => [
             'enablePrettyUrl' => true,
             'enableStrictParsing' => true,
             'showScriptName' => false,
             'rules' => [
-
                 /*
-                 * Status route
+                 * Status
                  */
                 'GET /site/system-status' => 'site/system-status',
 
                 /*
-                 * Catch all to throw 401 or 405
```

Comment:
Re: our conversation about throwing a 401 vs. a 404 for an unauthorized request to a non-existent endpoint, this may be an example of how to do that.